scripts/align-version.sh

```diff
@@ -9,11 +9,11 @@ scriptdir=$(cd $(dirname $0) && pwd)
 # go to repo root
 cd ${scriptdir}/..
 
-files="lerna.json $(find . -name package.json | grep -v node_modules | grep -v "^./package.json" | xargs)"
+files="$(find . -name package.json | grep -v node_modules | xargs)"
```

Comment:
Hmm. I guess I had assumed this would just be a `lerna version` command if we put the version in `lerna.json`. Isn't it?

Reply:
Lerna version doesn’t take care of peer dependencies and occasionally misses things. This is more straightforward I believe.